domain/src/main/java/edu/nextstep/camp/calculator/domain/MemoryHistorySaver.kt

```diff
@@ -0,0 +1,14 @@
+package edu.nextstep.camp.calculator.domain
+
+class MemoryHistorySaver : ExpressionHistorySaver {
```

Comment:
수정 요청사항으로 드리는 부분은 아닙니다만 기회가 되신다면 MemoryHistorySaver도 
Expression 클래스처럼 불변객체로 구현해보시기를 권해드립니다. 👀

```suggestion
data class MemoryHistory(val items: List<Item> = emptyList()) {

    operator fun plus(value: Item) = MemoryHistory(items + value)

    companion object {
        val EMPTY = MemoryHistory()
    }

    data class Item(val expression: String, val result: Int)
}
```